requirements/partial/requirements_production.txt

```diff
@@ -16,3 +16,4 @@ opentelemetry-instrumentation-requests
 opentelemetry-instrumentation-flask
 protobuf==3.20.1
 Babel==2.10.2
+psutil==5.9.1
```

Comment:
won't be needed if ram usage will no longer be reported in `openslides_backend/action/action_worker.py`

Reply:
removed